### ISSUE TITLE
Broken reference for "Appropriate Use of Well-Known URIs"

### ISSUE THREAD

**Issue by psimonyi** [WICG/change-password-url#26]
In section 3:
> Servers must not locate the actual change password page at the change password url, per RFC8615 §1.1 Appropriate Use of Well-Known URIs.

In PR #17, the reference was updated from the obsolete [RFC 5785](https://tools.ietf.org/html/rfc5785) to the current [RFC 8615](https://tools.ietf.org/html/rfc8615), leaving the section reference unchanged.  However, 8615 does not contain a section 1.1, nor does it contain any section directly addressing the appropriate use of well-known URIs.

The best reference I can find in 8615 to support this spec prohibiting servers from locating the resource itself at the well-known URI is section 1, in which the RFC implies that well-known URIs are for metadata (only?): the penultimate paragraph includes "Future specifications that need to define a resource *for such metadata* [...]".

See also issue #9.

**Comment by FaRrules** [comment]
Sorry dont get it i cant hack and this stuff i Can make thing and Waek spots com in light

Sendt fra min iPhone

**Comment by ferdnyc** [comment]
Indeed, in fact my interpretation of RFC 8615 (in its final form) is that it explicitly _disclaims_ any prescriptive statements regarding the content of resources located at well-known URIs.

> At a minimum, a registration will reference a specification that defines the format and associated media type(s) to be obtained by dereferencing the well-known URI

> Note that this specification defines neither how to determine the hostname to use to find the well-known URI for a particular application, nor the scope of the metadata discovered by dereferencing the well-known URI; both should be defined by the application itself.

The second quote adds even more support to @psimonyi's argument that well-known URIs are intended only for metadata. Nevertheless, it feels like, if the change-password spec wants to prohibit serving actual password-change interfaces directly from the well-known URI, RFC 8615 doesn't really require that; in fact, it explicitly refuses to make any such requirements. It feels like this sentence of the spec needs some adjustment (beyond the reference to a nonexistent section of the RFC):

> Servers must not locate the actual change password page at the [change password url](https://www.w3.org/TR/2024/WD-change-password-url-20240603/#change-password-url), per [RFC8615 §1.1 Appropriate Use of Well-Known URIs](https://www.rfc-editor.org/rfc/rfc8615#section-1.1).

While RFC 8615 doesn't really support that restriction (again, in my interpretation), it _does_ explicitly delegate the **power** to impose such requirements to the change-password spec, by giving it exclusive authority to define "the scope of the metadata discovered by dereferencing the well-known URI". If the change-password spec defines the information served by `/.well-known/change-password` not as a password-change page, but a redirect to one, then that's what it must be.

(That being said, allowing `/.well-known/change-password` to resolve as an HTML document containing a refresh pragma -- while sensible -- does complicate matters somewhat. If not for that, a result code 200 web document could be prohibited outright as a response.)
